config.json

```diff
@@ -709,7 +709,10 @@
       "core": true,
       "difficulty": 6,
       "slug": "binary-search",
-      "topics": null,
+      "topics": [
+        "recursion",
+        "arrays"
```

Comment:
Good choices! I'd recommend adding `searching` here too, but I think that's the only change needed!

Reply:
I would add `generics` as well :)